### ISSUE TITLE
Plans to use the new edge service port number

### ISSUE THREAD

**Issue by yonahforst** [localstack/serverless-localstack#99]
Are there plans to use the new edge service for configuring endpoints? The reason I'm asking is because I want to pass in the port number(s) from an environment variable but I don't want to have to specify one for each service, just the edge port.

So something like this:
```yml
  localstack:
    host: http://${env:LOCALSTACK_HOST, 'localhost'}
    edgePort: ${env:LOCALSTACK_EDGE_PORT, '4566'}
    stages:
      - local
```

**Comment by whummer** [comment]
Merged #100 - thanks @yonahforst . We'll release a new version shortly.

**Comment by whummer** [comment]
Done - released `serverless-localstack@0.4.25` to npm.js
